kiwi/storage/setup.py

```diff
@@ -312,13 +312,13 @@ def _accumulate_volume_size(self, root_mbytes):
                 [size_type, req_size] = volume.size.split(':')
                 disk_add_mbytes = 0
                 if size_type == 'freespace':
-                    disk_add_mbytes += int(req_size) + \
-                        Defaults.get_min_volume_mbytes()
+                    disk_add_mbytes += int(req_size)
```

Comment:
why did you remove the additional min bytes from the freespace setup ? The current use in bytes plus the requested free size will not lead the requested size being really free due to the volume overhead. I rather have a little bit more free than less than requested. Thoughts ?